pkg/ui/src/FormFields/TextField/TextField.svelte

```diff
@@ -61,6 +64,6 @@
 		{/if}
 	</div>
 	{#if helpText}
-		<p class={helpTextClasses}>{helpText}</p>
+		<p on:keypress={onHelpTextClick} on:click={onHelpTextClick} class={helpTextClasses}>{helpText}</p>
```

Comment:
This component is definitely overloaded with responsibility now. Can we define a "helpText" slot and manage all of this in the book form itself? Keep only the `error` prop (renamed to `isValid`) for now so that the rest of the component knows how to style itself when there's a validation error